bepdf/beos/FileInfoWindow.cpp

```diff
@@ -87,7 +87,7 @@ static int ToInt(const char *s, int i, int n) {
 static const char *ToDate(const char *date, time_t *time) {
 static char s[80];
 	struct tm d;
-	memset(&d, sizeof(d), 0);
+    memset(&d,0, sizeof(d));
```

Comment:
```suggestion
    memset(&d, 0, sizeof(d));
```

Reply:
The formatting, yes ? I overlooked that, your editing is better.